addons/web_tour/static/src/js/tour_service.js

```diff
@@ -36,6 +36,12 @@ return session.is_bound.then(function () {
         const disabled = session.tour_disable || device.isMobile;
         var tour_manager = new TourManager(rootWidget, consumed_tours, disabled);
 
+        const iframeContainer = 'wysiwyg_iframe';
+        const isInIframe = window.frameElement && window.frameElement.closest(`.${iframeContainer}`);
+        if (isInIframe) {
+            return tour_manager;
+        }
```

Comment:
I really don't know what this code is about. Was a comment really not needed here?

Reply:
That's a workaround to avoid running the tours when they are loaded inside an iframe. 
But probably that the tests assets should never be served in the context of an iframe, I can investigate that furhter